mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/model/openjpa/AbstractJPAMailboxMessage.java

```diff
@@ -59,7 +59,6 @@
 import org.apache.james.mailbox.store.mail.model.Property;
```

Comment:
Not sure where that one is used though....

I think we can rely on default method to avoid getting this "comparable" behaviour implementation specific....